### ISSUE TITLE
Building Packer on S390x

### ISSUE THREAD

**Issue by james-crowley** [hashicorp/packer#7560]
I am trying to build Packer on s390x. I pulled down the latest release, v1.4.0, and ran `make bin`, just to test out if a build would even work. 

As of right now it seems to fail. Here is part of the output:

```
WARN: 'make bin' is for debug / test builds only. Use 'make release' for release builds.
==> Checking for necessary tools...
==> Entering Packer source dir...
==> Ensuring output directories are present...
==> Removing old builds...
==> Building...
Number of parallel builds: 3

-->     linux/s390x: github.com/hashicorp/packer

1 errors occurred:
--> linux/s390x error: exit status 2
Stderr: # github.com/hashicorp/packer/vendor/github.com/creack/goselect
vendor/github.com/creack/goselect/fdset.go:14:14: undefined: NFDBITS
vendor/github.com/creack/goselect/fdset.go:19:14: undefined: NFDBITS
vendor/github.com/creack/goselect/fdset.go:24:21: undefined: NFDBITS

==> Copying binaries for this platform...

==> Results:
total 0
```

I opened an issue, https://github.com/creack/goselect/issues/14 , cause I think the error above has to do with `goselect` library. But I am not familiar enough with Go to fully understand whats going on and not 100% sure its all that libraries fault.

I was able to port terraform okay, but ran into some vendor errors along the way, hence I expect there is going to be some bugs with Packer.

**Comment by james-crowley** [comment]
@azr Do you have a recommended version of GO I should be using? Currently I am using 1.11.5.

**Comment by azr** [comment]
Hey @james-crowley thanks for opening, let's see what happens in https://github.com/creack/goselect/issues/14 before trying to look for fixes.

``` bash
GO111MODULE=on go mod why github.com/creack/goselect          
# github.com/creack/goselect
github.com/hashicorp/packer/builder/scaleway
github.com/scaleway/scaleway-cli/pkg/api
github.com/scaleway/scaleway-cli/pkg/utils
github.com/moul/gotty-client
github.com/creack/goselect
```

I think changing version shouldn't change much. Though I would recommend on going over to go 1.12 which has a better go modules support. I'm currently running on `go1.12.4`. I think this won't change much to this issue.

**Comment by SwampDragons** [comment]
This was closed in #7567
